lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -0,0 +1,134 @@
+require 'active_support/core_ext/module/delegation'
+require 'pathname'
+
+module ManageIQ
+  module ApplianceConsole
+    class MessageConfiguration
+      attr_reader :username, :password,
+                  :miq_config_dir_path, :config_dir_path, :sample_config_dir_path,
+                  :tools_log4_properties_path, :tools_log4_properties_sample_path,
+                  :client_properties_path,
+                  :keystore_dir_path, :truststore_path, :keystore_path,
+                  :messaging_yaml_sample_path, :messaging_yaml_path
+
+      BASE_DIR                          = "/opt/kafka".freeze
+      LOGS_DIR                          = "#{BASE_DIR}/logs".freeze
+      CONFIG_DIR                        = "#{BASE_DIR}/config".freeze
+      SAMPLE_CONFIG_DIR                 = "#{BASE_DIR}/config-sample".freeze
+      MIQ_CONFIG_DIR                    = "/var/www/miq/vmdb/config".freeze
+
+      def initialize(options = {})
+        @username, @password = nil
+
+        @username = options[:username] || "admin"
+        @password = options[:password]
+
+        @miq_config_dir_path               = Pathname.new(MIQ_CONFIG_DIR)
+        @config_dir_path                   = Pathname.new(CONFIG_DIR)
+        @sample_config_dir_path            = Pathname.new(SAMPLE_CONFIG_DIR)
+
+        @tools_log4_properties_path        = config_dir_path.join("tools-log4j.properties")
+        @tools_log4_properties_sample_path = sample_config_dir_path.join("tools-log4j.properties")
+        @client_properties_path            = config_dir_path.join("client.properties")
+
+        @keystore_dir_path                 = config_dir_path.join("keystore")
+        @truststore_path                   = keystore_dir_path.join("truststore.jks")
+        @keystore_path                     = keystore_dir_path.join("keystore.jks")
+
+        @messaging_yaml_sample_path        = miq_config_dir_path.join("messaging.kafka.yml")
+        @messaging_yaml_path               = miq_config_dir_path.join("messaging.yml")
+      end
+
+      def create_tools_log_config
+        say(__method__.to_s.tr("_", " ").titleize)
+
+        FileUtils.cp(tools_log4_properties_sample_path, tools_log4_properties_path) unless file_found?(tools_log4_properties_path)
```

Comment:
Did we miss this in the RPM spec?  If so, I'd prefer to fix it there.

Reply:
@bdunne Good idea.
Addressed by: https://github.com/ManageIQ/manageiq-rpm_build/pull/122
and commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/130/commits/8545e96290fa0369a7693fb7f2f8cfb3824b49a5